code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -78,6 +79,24 @@
 		"is twisting their own neck!",
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/tail_lash
+		)
+
+/mob/living/carbon/human/proc/tail_lash()
```

Comment:
Does this have a cooldown? I wonder if you can macro it and lash someone machine-gun style.  Also- perhaps a sound effect would be well applied here?

Reply:
i put in a stamina cost. so it drains 15 staimina on use. i forgot the sound effect.

Reply:
Still wack being able to slap people as fast as you can get the attacks off even if it'll stun you after a few strikes. Any cooldown is better than none